charts/gateway-operator/UPGRADE.md

```diff
@@ -0,0 +1,24 @@
+# Upgrade considerations
+
+New versions of this chart may add significant new functionality or
+deprecate/entirely remove old functionality. This document covers how and why
+users should update their chart configuration to take advantage of new features
+or migrate away from deprecated features.
+
+In general, breaking changes deprecate their old features before removing them
+entirely. While support for the old functionality remains, the chart will show
+a warning about the outdated configuration when running
+`helm install/status/upgrade`.
+
+Note that not all versions contain breaking changes. If a version is not
+present in the table of contents, it requires no version-specific changes when
+upgrading from a previous version.
+
+## Updates to CRDs
+
+Helm installs CRDs at initial install but [does not update them after][hip0011].
+Some chart releases include updates to CRDs that must be applied to successfully
+upgrade. Because Helm does not handle these updates, you must manually apply
```

Comment:
I think that what users will look for here is how to upgrade CRDs. How about adding some incantations that can be used to upgrade?

Reply:
Done in 6743b05bfaf79be9735161a1f905c6b7df80c313